src/olympia/reviewers/utils.py

```diff
@@ -319,7 +319,9 @@ class HeldDecisionQueueTable:
 
     @classmethod
     def get_queryset(cls, request, **kwargs):
-        return ContentDecision.objects.filter(action_date=None).order_by('created')
+        return ContentDecision.objects.filter(
+            action_date=None, overridden_by=None
```

Comment:
So if the decision has been overridden in Cinder, we consider it doesn't need second level approval ?

Reply:
Yes.  It still won't have an action date (so we know that the disable/reject wasn't carried out, and no emails were sent), but it doesn't need any further review.

Reply:
Could you refactor this in its own queryset/manager method on `ContentDecision`, like `held_for_second_approval()` or something, adding a short docstring with that info?

Reply:
https://github.com/mozilla/addons-server/pull/23083/commits/42b4821208de59d36b61b50e567430bec927905a